### PR TITLE
Wbi resize disk

### PR DESCRIPTION
**Pull request by bcreddy-gcp**
**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
workbench: Added support for resizing disk
```

### CODE REVIEW

File: mmv1/products/workbench/Instance.yaml [hunk 1]
```diff
@@ -270,12 +270,12 @@ properties:
         name: bootDisk
         default_from_api: true
         description: The definition of a boot disk.
-        immutable: true
+        update_url: 'projects/{{project}}/locations/{{location}}/instances/{{name}}/resizeDisk'
```
Comment (rileykarson) [inline mmv1/products/workbench/Instance.yaml:273]: This is currently generating a null update- I think we need to apply this to each of the leaf fields we intend to update.

**Comment by bcreddy-gcp** [comment]
https://github.com/GoogleCloudPlatform/magic-modules/pull/10972

File: mmv1/products/workbench/Instance.yaml (hunk 1 above)
Comment (zli82016) [reply mmv1/products/workbench/Instance.yaml:273]: It looks like the update_url only works for root properties. https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/templates/terraform/resource.erb#L821

Custom code is used to update disks in another [PR](https://github.com/GoogleCloudPlatform/magic-modules/pull/10972/files), as updating disks needs a different method `ResizeDisk`.  @rileykarson , does custom code sound reasonable to handle this case?

**Comment by zli82016** [comment]
It looks like the custom code works to resize disks in https://github.com/GoogleCloudPlatform/magic-modules/pull/10972. Closing this PR.

**Pull request closed without merging**